### ISSUE TITLE
Nodes pagination not working

### ISSUE THREAD

**Issue by syrusakbary** [facebook/relay#40]
I've followed the [setVariables example](https://facebook.github.io/relay/docs/api-reference-relay-container.html#setvariables) for creating a pagination in nodes, however is not working with the current changes: https://github.com/syrusakbary/relay-starter-kit/commit/75703bb041772fe6c2592ca198af38d0166fb5e6

Here is the error I'm getting in the browsers console:
```
Uncaught Error: Invariant Violation: RelayQuery: Field `widgets` does not have an argument `after`.
```

Related issue in *Relay Starter Kit*: https://github.com/facebook/relay-starter-kit/issues/3

**Comment by steveluscher** [comment]
Thanks! We're on it.

**Comment by josephsavona** [comment]
FYI we fixed this upstream and will be syncing the change down later today or early tomorrow.

**Comment by syrusakbary** [comment]
Great! Thanks @josephsavona @steveluscher

**Comment by josephsavona** [comment]
This was fixed in 0.1.1 - https://github.com/facebook/relay/blob/master/CHANGELOG.md#011-august-14-2015

Thanks for reporting, and feel free to reopen if there are any further issues!
